app/components/dashboard/dashboard.component.js

```diff
@@ -61,15 +62,28 @@ class Dashboard extends React.Component {
     this.getRegion();
 
     getActivities();
+    getTeamsList();
     window.addEventListener("resize", this.measure);
   }
 
   componentWillReceiveProps(nextProps) {
     const { breakdown } = nextProps;
+    const { teams } = nextProps;
+
     if (breakdown.offices && breakdown.offices.length) {
       const statistics = keyBy(breakdown.offices, "name");
       this.setState({ statistics });
     }
+
+    if (teams && teams.length > 0) {
+      const teamsList = _.map(teams, (team, index) => ({
+        key: team.name,
+        text: team.name,
+        value: team.name,
+      }));
+
+      this.setState({ teamsList });
```

Comment:
Do we need teamsList? I can't see where you're using it in dashboard component? 
You are setting all fields key, text, value the same team.name, is that correct?

Reply:
I believe I removed all references to teamsList in dashboard.component.js!